utils/src/server/helpers.rs

```diff
@@ -51,12 +112,12 @@ pub fn send_response(id: Id, tx: &MethodSink, result: impl Serialize) {
 }
 
 /// Helper for sending JSON-RPC errors to the client
-pub fn send_error(id: Id, tx: &MethodSink, error: ErrorObject) {
-	let json = match serde_json::to_string(&RpcError { jsonrpc: TwoPointZero, error, id }) {
-		Ok(json) => json,
+pub fn send_error(id: Id, tx: &MethodSink, error: ErrorObject, max_response_size: u32) {
```

Comment:
Not sure this is necessary, it should be pretty safe to assume that errors never exceed whatever limit people have set, plus if they have set the limit to, like, `10`, it would be nice to ignore it to send the error that we've exceeded the limit of `10` :D.